spec/system/notifications/index_spec.rb

```diff
@@ -12,19 +12,40 @@
     let(:notification_message) { "#{volunteer.display_name} resolved a follow up. Click to see more." }
     let!(:followup) { create(:followup, creator: admin, case_contact: case_contact) }
 
-    it "lists my notifications" do
+    before do
       sign_in volunteer
 
       visit case_contacts_path
       click_button "Resolve"
+    end
 
+    it "lists my notifications" do
       sign_in admin
       visit notifications_path
 
       expect(page).to have_text(notification_message)
       expect(page).to have_text("Followup resolved")
       expect(page).not_to have_text(I18n.t(".notifications.index.no_notifications"))
     end
+
+    context "when volunteer changes its name" do
```

Comment:
Please add a test for old notifications being displayable

Reply:
Not sure we can do that, as notifications fail if required params are not filled on save. I'll see what i can do